extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/TaskStartTimeoutFault.java

```diff
@@ -32,13 +32,13 @@ public class TaskStartTimeoutFault extends BaseMSQFault
 {
   static final String CODE = "TaskStartTimeout";
 
-  private final int numTasksNotStarted;
+  private final int pendingTasks;
   private final int totalTasks;
   private final long timeout;
 
   @JsonCreator
   public TaskStartTimeoutFault(
-      @JsonProperty("numTasksNotStarted") int numTasksNotStarted,
+      @JsonProperty("pendingTasks") int pendingTasks,
```

Comment:
This warrants a change in the docs at `druid/docs/multi-stage-query/reference.md`.

Reply:
Thanks! Changed.